### ISSUE TITLE
Cell sizes are very large

### ISSUE THREAD

**Issue by DaelonSuzuka** [peterhoglund/gd-sheets#1]
This plugin installed with no visible warnings or errors, but the cells are all huge.

I couldn't find any option to adjust size or spacing.

![Godot_v3 4 2-stable_win64_UOYsThP9kW](https://user-images.githubusercontent.com/18042232/152297122-77d6833b-983b-45ba-9d68-52b64615ea01.png)

**Comment by peterhoglund** [comment]
What screen resolution do you have? I think I know why this is happening. I am defining two sizes for the cells depending on screen size to adjust for HiDPI screens. But it's a very naive implementation atm. I think you are getting the larger sizes but with the smaller resolution.

**Comment by DaelonSuzuka** [comment]
24" 2560x1440p. I have a 4k monitor as well, let me fire this up again and check it on that.

edit: I can't figure out how to make godot start on the other monitor... I'll try to duplicate this some other way.

**Comment by peterhoglund** [comment]
I found an error in the function that sets the sizes. I've updated the files, can you try to download the addon again and see it solved it?

**Comment by DaelonSuzuka** [comment]
Sorry, no luck. I was able to use [Godot Plugin Refresher](https://github.com/godot-extended-libraries/godot-plugin-refresher) to force refresh gdsheets on my different monitors, including a 4k one. 

If I force refresh it on the 10" 1920x1200 monitor, it loads the small cells, which then looks good on my other monitors (24" 1440p and 39" 4k). I don't think I'd ever want the double size cells with these screens.

How about an option to just disable hidpi scaling?

**Comment by peterhoglund** [comment]
Yes, that might be the simplest solution. I think I will also include a setting for default cell size.
